### PR TITLE
MWS-4620: avoid race condition for Tag-filter and URL-mapping

### PR DESCRIPTION
**Pull request by polmih**
In Action manager, selectInputAct, trigger `wb-contentupdated` after the filters changes are applied via URL mapping.[
AEM test page issue example](https://authorca1.dev.canada.adobecqms.net/content/canadasite/en/auditor-general/our-work/audit-reports.html?rt=ag&y=2024)

### CODE REVIEW

**Review by Garneauma — CHANGES_REQUESTED** [review]
@polmih We'll need a working example to test that use case. You can add it in tagfilter/tagfilter-edge-cases-en|fr.html

File: components/wb-actionmng/actionmng.js [hunk 1]
```diff
@@ -430,6 +430,8 @@ var $document = wb.doc,
 				input.setAttribute( "checked", true );
 			}
 		} );
+		const tagfilterContainer = sourceElm.closest( ".wb-tagfilter" ) || sourceElm;
+		$( tagfilterContainer ).trigger( "wb-contentupdated" );
```
Comment (Garneauma) [inline components/wb-actionmng/actionmng.js]: No need to get the tagfilter element. The tagfilter update is triggered if any child element sends the "wb-contentupdated" event.
```suggestion

		$( sourceElm ).trigger( "wb-contentupdated" );
```

File: components/wb-urlmapping/selectinput-en.html [hunk 2]
```diff
@@ -222,3 +223,251 @@ <h2>Example without source</h2>
...
+	"section": ".events-list3",
+	"uiTemplate": "#searchFilter3"
+}' data-wb-paginate='{
+	"itemsPerPage": 5,
+	"section": ".events-list3",
+	"selector": "li",
+	"uiTarget": "#filteredEventsListPagination3"
+}'>
+	<h2>Example with Tag-filter component</h2>
+	<div id="searchFilter3" class="row mrgn-tp-lg">
```
Comment (Garneauma) [inline components/wb-urlmapping/selectinput-en.html:246]: Please either add an instruction or a link on how to test this example. For example: "Add the following parameter in the url..."

Comment (polmih) [reply components/wb-urlmapping/selectinput-en.html:246]: There is a link on the top of the page

File: components/wb-actionmng/actionmng.js (hunk 1 above)
Comment (polmih) [reply components/wb-actionmng/actionmng.js]: Re-tested on local AEM with positive outcome

File: components/wb-urlmapping/selectinput-fr.html [hunk 3]
```diff
@@ -222,3 +223,288 @@ <h2>Exemple sans <code>source</code></h2>
...
+}'>
+	<h2>Exemple avec le composant de filtres de balise</h2>
+	<p>Ajoutez le paramètre suivant dans l’URL :</p>
+	<ul>
+		<li><code>lng=fr</code> pour le français ou </li>
+		<li><code>lng=eng</code> pour l’anglais </li>
+		<li><code>loc=mtl</code> pour Montréal</li>
+		<li><code>loc=otw</code> pour Ottawa</li>
+		<li><code>loc=tor</code> pour Toronto</li>
+	</ul
```
Comment (Garneauma) [inline components/wb-urlmapping/selectinput-fr.html]: Broken closing tag.
```suggestion
	</ul>
```

File: components/wb-urlmapping/selectinput-fr.html [hunk 4]
```diff
@@ -222,3 +223,288 @@ <h2>Exemple sans <code>source</code></h2>
...
+}' data-wb-paginate='{
+	"itemsPerPage": 5,
+	"section": ".events-list3",
+	"selector": "li",
+	"uiTarget": "#filteredEventsListPagination3"
+}'>
+	<h2>Exemple avec le composant de filtres de balise</h2>
+	<p>Ajoutez le paramètre suivant dans l’URL :</p>
+	<ul>
+		<li><code>lng=fr</code> pour le français ou </li>
```
Comment (Garneauma) [inline components/wb-urlmapping/selectinput-fr.html]:
```suggestion
		<li><code>lng=fr</code> pour le français</li>
```

File: components/wb-urlmapping/selectinput-en.html [hunk 5]
```diff
@@ -222,3 +223,267 @@ <h2>Example without source</h2>
...
+}' data-wb-paginate='{
+	"itemsPerPage": 5,
+	"section": ".events-list3",
+	"selector": "li",
+	"uiTarget": "#filteredEventsListPagination3"
+}'>
+	<h2>Example with Tag-filter component</h2>
+	<p>Add the following parameter in the url:</p>
+	<ul>
+		<li><code>lng=fr</code> for French or </li>
```
Comment (Garneauma) [inline components/wb-urlmapping/selectinput-en.html]:
```suggestion
		<li><code>lng=fr</code> for French</li>
```

File: components/wb-urlmapping/selectinput-fr.html [hunk 6]
```diff
@@ -23,6 +23,7 @@
 	<li><a href="?">Vide <code>?</code></a></li>
 	<li><a href="?f=b">Bananes <code>?f=b"</code></a></li>
 	<li><a href="?f=g">Raisins <code>?f=r"</code></a></li>
+	<li><a href="?lng=eng&loc=otw">Anglais en Ottawa <code>?lng=eng&loc=otw"</code></a></li>
```
Comment (Garneauma) [inline components/wb-urlmapping/selectinput-fr.html]:
```suggestion
	<li><a href="?lng=eng&loc=otw##filteredEventsList3">En anglais à Ottawa <code>?lng=eng&loc=otw"</code></a></li>
```

File: components/wb-urlmapping/selectinput-en.html [hunk 7]
```diff
@@ -23,6 +23,7 @@
 	<li><a href="?">Empty <code>?</code></a></li>
 	<li><a href="?f=b">Bananas <code>?f=b"</code></a></li>
 	<li><a href="?f=g">Grapes <code>?f=g"</code></a></li>
+	<li><a href="?lng=eng&loc=otw">English in Ottawa <code>?lng=eng&loc=otw"</code></a></li>
```
Comment (Garneauma) [inline components/wb-urlmapping/selectinput-en.html]:
```suggestion
	<li><a href="?lng=eng&loc=otw#filteredEventsList3">In English in Ottawa <code>?lng=eng&loc=otw"</code></a></li>
```
